templates/github/.github/ISSUE_TEMPLATE/feature_request.md.j2

```diff
@@ -0,0 +1,20 @@
+---
+name: ✨ Feature request
+about: Suggest an idea for this project
+title: ''
+labels: Feature, Triage-Needed
```

Comment:
A feature request should not be marked as `Triage-Needed` as well. Am I correct?